release-notes/rn-axon-framework/rn-af-major-releases.md

```diff
@@ -2,69 +2,78 @@
 
 All the enhancements and features which have been introduced to our major releases of the Axon Framework are noted here.
 
-## Release 4.3
+## Release 4.4
 
 ### _Enhancements_
 
-* Aggregate Polymorphism has been introduced, allowing for an aggregate hierarchy as would come natural from a domain model.
+* Axon Framework can now be used in conjunction with [Spring Boot Developer Tools](https://docs.spring.io/spring-boot/docs/1.5.16.RELEASE/reference/html/using-boot-devtools.html).
+  You can simply achieve this by adding the required dev-tools dependency to your project.
 
-  To set this up, the `AggregateConfigurer#withSubtypes(Class... aggregates)` method can be used.
+* As a partial solution to [#1106](https://github.com/AxonFramework/AxonFramework/issues/1106), Axon Server can now be used to schedule events.
+  Building an `AxonServerEventScheduler` as the `EventScheduler` implementation as defined through the builder is sufficient to start with scheduling events through Axon Server.
+  
+* An `EventTrackerStatusChangeListener` can now be configured for a `TrackingEventProcessor`, as was request in [#1338](https://github.com/AxonFramework/AxonFramework/issues/1338).
+  It can be configured through the `TrackingEventProcessorConfiguration`, allowing users to react upon status changes of each thread processing events.
```

Comment:
```suggestion
  It can be configured through the `TrackingEventProcessorConfiguration`, allowing users to react upon status changes of each thread processing event.
```

Reply:
This is actually intended as plural, as a thread will handle several events. I'll change it up to "event messages" to be clearer on this fact.